packages/ui/src/components/timeline/RangeSelector.tsx

```diff
@@ -78,13 +79,46 @@ export function RangeSelector({rangeRef}: RangeSelectorProps) {
           onDrop();
         }
       }}
-      onDblClick={() => {
-        meta.shared.range.update(0, Infinity, duration, fps);
-      }}
     >
-      <RangeHandle value={start} setValue={setStart} onDrop={onDrop} />
-      <div class={styles.handleSpacer} />
-      <RangeHandle value={end} setValue={setEnd} onDrop={onDrop} />
+      <div
+        style={{
+          flexDirection: start > end ? 'row-reverse' : 'row',
+          left: `${framesToPercents(Math.ceil(Math.max(0, normalizedStart)))}%`,
+          right: `${
+            100 - framesToPercents(Math.ceil(Math.min(duration, normalizedEnd)))
+          }%`,
+        }}
+        className={clsx(
+          styles.range,
+          shiftHeld && !controlHeld && styles.active,
```

Comment:
When holding `CTRL` + `SHIFT` the range should still be highlighted with the handles remaining invisible. Right now there's no indication that anything can be done with the range.

Reply:
Perhaps we can find a way to highlight the whole track because all of it can be edited, not just the current range. I think this might be a little more intuitive, don't you?

Reply:
Because as of right now, you can also draw the new range "through" the current range, which I think is convenient. But highlighting the range and then manipulating the underlying component feels even more confusing to me then giving no indication (apart from the changing cursor, but that is obviously only visible when hovering the range track..). I'm not quite sure what would be the best solution here

Reply:
I phrased it poorly. What I meant is that by default both the range and the audio track are greyed out because they are not editable. Holding `SHIFT` is meant to enter an edit mode where both of them can be easily edited without mistakenly moving the playhead. This is why when `SHIFT` is held both of these things become lighter - to symbolize that they are no longer disabled. Holding `SHIFT` + `CTRL` should still visually act like we're in edit mode.

Reply:
Oh okay, that makes sense!